static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -126,27 +134,34 @@ require.def(
                     self.tvmwPlugin.SetSource(self.originalSource);
                 }, false);
             },
+            
+            _openPlayerPlugin : function() {
+                if (this._currentPlayer !== undefined) {
+                    this._playerPlugin.Close();
+                }
+                this._playerPlugin.Open("Player", "1.010", "Player");
+                this._currentPlayer = this.PlayerEmps.Player;
+            },
+            
+            _openStreamingPlayerPlugin : function() {
+                if (this._currentPlayer !== undefined) {
+                    this._playerPlugin.Close();
+                }
+                this._playerPlugin.Open("StreamingPlayer", "1.0", "StreamingPlayer");
+                this._currentPlayer = this.PlayerEmps.StreamingPlayer;
+            },
+            
+            _closePlugin: function() {
+                this._playerPlugin.Close();
+                this._currentPlayer = undefined;
+            },
 
             _initPlayer: function(source) {
-                this.logger.info('Calling this.playerPlugin.Execute("InitPlayer", ' + this._source + ')');
                 var result = this._playerPlugin.Execute("InitPlayer", source);
-                if (result === -1) {
-                    this._toError('STOP');
-                    return;
+                
+                if (result !== 1) {
+                    this._toError("Failed to initialize video: " + this._source);
                 }
-                this.logger.info('InitPlayer responded with ' + result);
-
-                this.logger.info('Calling this.playerPlugin.Execute("SetTotalBufferSize", ' +  32 *1024*1024 + ')');
-                result = this._playerPlugin.Execute("SetTotalBufferSize", 32 *1024*1024);
```

Comment:
Ignore this - just re-read your comments and saw the note that this is only for UHD.